Dockerfile-18

```diff
@@ -23,8 +23,7 @@ RUN apt-get -qy install \
       autoconf \
       libtool \
       debootstrap \
-      systemd \
-      ccache
```

Comment:
Why did you remove `ccache`?

Reply:
Because I added it in my efforts to speed up package build performance. I was not able to discern that it had any demonstrable impact on compile times during the build step.